browser/widevine/brave_widevine_bundle_unzipper.cc

```diff
@@ -22,8 +24,10 @@ namespace {
 bool IsWidevineCdmFile(const base::FilePath& file_path) {
   CHECK(!file_path.IsAbsolute());
   return base::FilePath::CompareEqualIgnoreCase(
-      file_path.value(),
-      base::GetNativeLibraryName(kWidevineCdmLibraryName));
+             file_path.value(),
```

Comment:
I think storing all files also would be fine because all they are informative and maybe they could be used in the future.
and they are very small size files.